shaders/src/post_process_inputs.fs

```diff
@@ -1,12 +1,24 @@
 LAYOUT_LOCATION(LOCATION_UVS) in highp vec2 vertex_uv;
 
-LAYOUT_LOCATION(0) out vec4 fragColor;
-
 /** @public-api */
 highp vec2 getUV() {
     return vertex_uv;
 }
 
 struct PostProcessInputs {
-    vec4 color;
+#if defined(OUTPUT0)
```

Comment:
Is it possible to not define any output? Do we error out somewhere in that case (at compilation time)?

Reply:
`MaterialBuilder` will add a default color output if no other ones are specified. Otherwise yes, we would error at shader compilation time (structs can't be empty in GLSL)

Reply:
👍